### PR TITLE
OBSDOCS-650: Add docs for CLF supporting clusterlogging

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OBSDOCS-650

Link to docs preview:
- https://68919--docspreview.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/cluster-logging-collector#log-collector-resources-scheduling_cluster-logging-collector

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/log-collector-resources-scheduling.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+= Configuring resources and scheduling for logging collectors
+
+Administrators can modify the resources or scheduling of the collector by creating a `ClusterLogging` custom resource (CR) that is in the same namespace and has the same name as the `ClusterLogForwarder` CR that it supports.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo} version 5.8 or newer.
+* You have created a `ClusterLogForwarder` CR.
+* The multi log forwarding feature is enabled on your cluster.
```
Comment (jcantrill) [inline modules/log-collector-resources-scheduling.adoc:16]: Consider striking this bullet all-together.

* There is no real action to "enable" mclf
* These statements as written apply whether it is the "legacy" deployment or mclf deployment

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: modules/log-collector-resources-scheduling.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: <name> <1>
+  namespace: <namespace> <2>
+spec:
+  managementState: Managed
+  collection:
+    type: vector
+# ...
```
Comment (anpingli) [inline modules/log-collector-resources-scheduling.adoc:35]: It is better to give an example as below
apiVersion: "logging.openshift.io/v1"
kind: "ClusterLogging"
metadata:
  name:  <name> <1>
  namespace: <namespace> <2>
spec:
  managementState: "Managed"
  collection:
    type: "vector"
    tolerations:
    - key: "logging"
      operator: "Exists"
      effect: "NoExecute"
      tolerationSeconds: 6000
    resources:
      limits:
        memory: 1Gi
      requests:
        cpu: 100m
        memory: 1Gi
    nodeSelector:
      collector: needed

**Review by anpingli — COMMENTED** [review]
/lgtm

File: modules/log-collector-resources-scheduling.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+. Create a `ClusterLogging` CR that supports your existing `ClusterLogForwarder` CR:
++
+.Example `ClusterLogging` CR YAML
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name:  <name> <1>
```
Comment (jeana-redhat) [inline modules/log-collector-resources-scheduling.adoc:29]: For the YAML callouts, [comment](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml) them out as below:
```suggestion
  name:  <name> # <1>
```

**Review by jeana-redhat — COMMENTED** [review]
Just one nit, otherwise looks perfect imo

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/log-collector-resources-scheduling.adoc (hunk 3 above)
Comment (abrennan89) [reply modules/log-collector-resources-scheduling.adoc:29]: Good call, I had forgotten about this!

Comment (jeana-redhat) [reply modules/log-collector-resources-scheduling.adoc]: Honestly hoping this round of squad duty finally cements it in my head for my own work :upside_down_face:

**Pull request merged**